src/main/java/at/hannibal2/skyhanni/features/nether/reputationhelper/dailykuudra/DailyKuudraBossHelper.kt

```diff
@@ -45,11 +44,11 @@ class DailyKuudraBossHelper(private val reputationHelper: CrimsonIsleReputationH
         if (!SkyHanniMod.feature.misc.crimsonIsleReputationHelper) return
 
         val message = event.message
-        if (message != "                               §r§6§lKUUDRA DOWN!") return
+        if (!message.contains("KUUDRA DOWN!") || message.contains(":")) return
 
         for (line in ScoreboardData.sidebarLines) {
-            pattern.matchMatcher(line) {
-                val tier = group("tier").toInt()
+            if(line.contains("Kuudra's") && line.contains("Hollow") && line.contains("(")){
```

Comment:
What was wrong with the regex? Is the chat message different for tier 4+5?

Reply:
For some reason it didn't work, it was probably for the color of the text messing up the regex or something, I couldn't find a way to fix that other than this

Reply:
It's broken for all kuudra tiers, I did my testing on tier 1 kuudra and once fixed it was working for all tiers.

I don't know why the regex wasn't working, I used that website aswell and it was working there but not in the code

Reply:
Time to start the debugger again